lib/hammer_cli_csv/roles.rb

```diff
@@ -84,9 +84,7 @@ def create_roles_from_csv(line)
             @existing_roles[name] = role['id']
           else
             print "Updating role '#{name}'..." if option_verbose?
-            @api.resource(:roles).call(:update, {
-                'id' => @existing_roles[name]
-            })
+            # Nothing to update
```

Comment:
Actually, in csv I merge the concept of "role" and "filter" into the higher level concept of "role." Filters, in Foreman, are never re-used between the role models. So each line of CSV modifies a role w/ filters but the role itself only has a "name" attribute which I don't allow changing. Hence, nothing to update in that section of code.